### PR TITLE
Live Showcases

### PR DESCRIPTION
**Pull request by andreasknoepfle**
This improves documentation for the project by a lot, by making it possible to showcase components directly from the docs:
Screenshot 2021-03-31 at 14 00 38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 [![Hex pm](http://img.shields.io/hexpm/v/bitstyles_phoenix.svg?style=flat)](https://hex.pm/packages/bitstyles_phoenix)
 [![Hex docs](http://img.shields.io/badge/hex.pm-docs-green.svg?style=flat)](https://hexdocs.pm/bitstyles_phoenix)
-[![License](https://img.shields.io/hexpm/l/bitstyles_phoenix?style=flat)](LICENSE)
+[![License](https://img.shields.io/hexpm/l/bitstyles_phoenix?style=flat)](./LICENSE.txt)
 [![CircleCI](https://circleci.com/gh/bitcrowd/bitstyles_phoenix.svg?style=shield)](https://circleci.com/gh/bitcrowd/bitstyles_phoenix)
-
```
Comment (planktonic) [inline README.md:7]: Was there a reason to remove this newline? I find it looks a little strange having the text flow after the build logos

File: lib/bitstyles_phoenix/components/button.ex [hunk 2]
```diff
@@ -26,17 +22,20 @@ defmodule BitstylesPhoenix.Components.Button do
       iex> safe_to_string ui_button("Save", type: "submit", class: "foo bar")
       ~s(<button class="a-button foo bar" type="submit">Save</button>)
 
-      iex> safe_to_string ui_button("Save", type: "submit", variant: "primary")
-      ~s(<button class="a-button a-button--primary" type="submit">Save</button>)
+      iex> safe_to_string ui_button("Save", type: "submit", variant: :ui)
+      ~s(<button class="a-button a-button--ui" type="submit">Save</button>)
+
+      iex> safe_to_string ui_button("Save", type: "submit", variant: [:ui, :danger], class: "foo")
```
Comment (planktonic) [inline lib/bitstyles_phoenix/components/button.ex:28]: 🤩 Amazing

File: lib/bitstyles_phoenix.ex [hunk 3]
```diff
@@ -1,5 +1,40 @@
...
+  """)
+
+  showcase("UI Danger", """
+    ui_button("Button", variant: [:ui, :danger])
+  """)
+
+  showcase("UI Danger", """
+    ui_button("Button", variant: [:ui, :danger])
+  """)
+
```
Comment (planktonic) [inline lib/bitstyles_phoenix.ex:34]: This button is duplicated and can be removed, I think

```suggestion
```

**Review by planktonic — APPROVED** [review]
So good

File: README.md (hunk 1 above)
Comment (andreasknoepfle) [reply README.md:7]: accidents :D

File: README.md [hunk 4]
```diff
@@ -2,9 +2,8 @@
...
 Basic helpers for [bitstyles](https://github.com/bitcrowd/bitstyles) for elixir phoenix projects.
```
Comment (andreasknoepfle) [inline README.md:7]:
```suggestion

Basic helpers for [bitstyles](https://github.com/bitcrowd/bitstyles) for elixir phoenix projects.
```

**Review by maltoe — COMMENTED** [review]
nice! 🥇

**Pull request merged**